app/graphql/types/user_error.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class UserError < BaseObject
+    description 'A user-readable error'
+
+    field :message, String, null: false,
+      description: 'A description of the error'
+
+    field :path, [String], null: true,
```

Comment:
Not sure if having just attribute name would be enough. On the other hand, this could be useful for nested attributes if we need them, so I think we can keep it as it is for now.